solutions/usaco-915.mdx

```diff
@@ -9,6 +9,44 @@ author: Sathvik Chundru
 
 <LanguageSection>
  
+<PySection>
+
+```py
+import sys
+sys.stdin = open('herding.in', 'r')
+sys.stdout = open('herding.out', 'w')
+cows = list(map(int, input().split()))
+cows.sort()
+a = cows[0]
+b = cows[1]
+c = cows[2]
+
+# Best scenario: the three elements are already
+# in order.
+if c == a + 2:
+	print(0)
+# If there is a difference by 2,
+# it can be solved in one move.
+# 3 5 9 -> 5 7 9
```

Comment:
ok uh make this comment on maybe 2 lines? and multiline comments on python
```py
"""
are usually something like this
idk it's just usually how i do them
"""
```

Reply:
cool, ill start using them 👍